testing.xml

```diff
@@ -1,9 +1,9 @@
-<!--<?xml version="1.0" encoding="UTF-8"?>-->
-<!--<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">-->
-<!--<suite name="BDD Test Suite" verbose="1" parallel="tests" thread-count="1" configfailurepolicy="continue">-->
-<!--    <test name="Login and Update Profile Test" annotations="JDK" preserve-order="true">-->
-<!--        <classes>-->
-<!--            <class name="TestRunner"/>-->
-<!--        </classes>-->
-<!--    </test>-->
-<!--</suite>-->
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I do not see testng dependency in your pom file

Reply:
@mobilman89
https://github.com/KurisuLim/CucumberHomework/blob/master/pom.xml